kuma/javascript/src/article.jsx

```diff
@@ -63,6 +65,21 @@ export default function Article({ document }: DocumentProps) {
                     : 'article text-content'
             }
         >
+            {document.locale !== locale && (
+                <div id="doc-pending-fallback" class="warning">
+                    <p>
+                        <bdi>
+                            {gettext(
+                                "You're reading the English version of this content, since no translation exists yet for this locale."
```

Comment:
Surprised that `es-lint` is not complaining about the `"` here. You could avoid that by doing

```suggestion
                                'You’re reading the English version of this content since no translation exists yet for this locale.'
```